### PR TITLE
docs(review): content review

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: dedibox-network/ip-failover/how-to/configure-multiip-virtualmac.mdx [hunk 1]
```diff
@@ -27,34 +27,34 @@ As long as they share the same virtual MAC, your failover IPs must be moved in a
...
 
-1. Click **Server** > **Server list** to display a list of your physical machines. 
+1. Click **Server** > **Server list** to display a list of your physical machines.
 2. Select the server you want to configure and click **Manage**. The servers' status page displays.
 3. Scroll down to the network configuration section and click **+ Info** next to the failover IP you want to configure. The advanced configuration displays.
-4. Click **Add virtual MAC**. The virtual MAC configuration displays. Click **Edit** to display the virtual MAC address page. 
-5. Click the cogwheel next to the failover IP address that you want to add to a multi-IP MAC address group. A pop-up menu displays. Click **Use an existing MAC address**. 
-6. Select the MAC address to clone in the pop-up and click **OK** to validate. 
+4. Click **Add virtual MAC**. The virtual MAC configuration displays. Click **Edit** to display the virtual MAC address page.
+5. Click the cogwheel next to the failover IP address that you want to add to a multi-IP MAC address group. A pop-up menu displays. Click **Use an existing MAC address**.
```
Comment (bene2k1) [inline dedibox-network/ip-failover/how-to/configure-multiip-virtualmac.mdx:34]:
```suggestion
5. Click the cogwheel icon next to the failover IP address that you want to add to a multi-IP MAC address group. A pop-up menu displays. Click **Use an existing MAC address**.
```

File: dedibox-network/ip-failover/how-to/configure-multiip-virtualmac.mdx [hunk 2]
```diff
@@ -27,34 +27,34 @@ As long as they share the same virtual MAC, your failover IPs must be moved in a
...
+  When you decide to delete a virtual MAC address associated with an IP address that shares the same virtual MAC with other IPs, only the connection between that particular IP address and the MAC is removed. The other IP addresses remain unaffected and are not modified.
 </Message>
 
-1. Click **Server** > **Server list** to display a list of your physical machines. 
+1. Click **Server** > **Server list** to display a list of your physical machines.
 2. Select the server you want to configure and click **Manage**. The servers' status page displays.
 3. Scroll down to the network configuration section and click **+ Info** next to the failover IP you want to configure. The advanced configuration displays.
 4. Click **Edit**. The virtual MAC configuration displays. 
-5. Click the cogwheel next to the IP address you want to configure. Then click **Delete virtual MAC addresss**. 
+5. Click the cogwheel next to the IP address you want to configure. Then click **Delete virtual MAC address**.
```
Comment (bene2k1) [inline dedibox-network/ip-failover/how-to/configure-multiip-virtualmac.mdx:55]:
```suggestion
5. Click the cogwheel icon next to the IP address you want to configure. Then click **Delete virtual MAC address**.
```

File: dedibox-network/ip-failover/how-to/create-virtual-mac.mdx [hunk 3]
```diff
@@ -35,42 +35,43 @@ If you start a virtual machine in bridge mode, protection will be activated on t
...
 
 ## How to add a virtual MAC address
 
-1. Click **Server** > **Server list** to display a list of your physical machines. 
+1. Click **Server** > **Server list** to display a list of your physical machines.
 2. Select the server you want to configure and click **Manage**. The servers' status page displays.
 3. Scroll down to the network configuration section and click **+ Info** next to the failover IP you want to configure. The advanced configuration displays.
-4. Click **Add virtual MAC**. The virtual MAC configuration displays. 
+4. Click **Add virtual MAC**. The virtual MAC configuration displays.
 5. Click the cogwheel next to the IP address you want to configure. Then select the type of MAC address and click **Add ... virtual MAC address**.
```
Comment (bene2k1) [inline dedibox-network/ip-failover/how-to/create-virtual-mac.mdx:48]:
```suggestion
5. Click the cogwheel icon next to the IP address you want to configure. Then select the type of MAC address and click **Add ... virtual MAC address**.
```

File: dedibox-network/ip-failover/how-to/create-virtual-mac.mdx [hunk 4]
```diff
@@ -35,42 +35,43 @@ If you start a virtual machine in bridge mode, protection will be activated on t
...
 </Message>
 
-1. Click **Server** > **Server list** to display a list of your physical machines. 
+1. Click **Server** > **Server list** to display a list of your physical machines.
 2. Select the server you want to configure and click **Manage**. The servers' status page displays.
 3. Scroll down to the network configuration section and click **+ Info** next to the failover IP you want to configure. The advanced configuration displays.
-4. Click **Edit**. The virtual MAC configuration displays. 
-5. Click the cogwheel next to the IP address you want to configure. Then click **Delete virtual MAC addresss**. 
+4. Click **Edit**. The virtual MAC configuration displays.
+5. Click the cogwheel next to the IP address you want to configure. Then click **Delete virtual MAC address**.
```
Comment (bene2k1) [inline dedibox-network/ip-failover/how-to/create-virtual-mac.mdx:72]:
```suggestion
5. Click the cogwheel icon next to the IP address you want to configure. Then click **Delete virtual MAC address**.
```

File: dedibox-network/ip-failover/how-to/configure-windows.mdx [hunk 5]
```diff
@@ -21,18 +21,15 @@ This page shows you how to configure a [failover IP](/dedibox-network/ip-failove
...
+2. Open the start menu and type **Ethernet settings**. The Ethernet window displays.
     <Lightbox src="scaleway_windows_ipfo1.webp" alt="" />
-3. Click **Change adatper options**. The network connections window displays. 
+3. Click **Change adatper options**. The network connections window displays.
 4. Right click on the Internet connection. Then click **Properties** in the pop-up menu.
 5. Select **Internet Protocol version 4 (TCP/IPv4)** in the list of connection items and click **Properties**.
 6. Enter the static configuration of the principal IP address of the server. Then click **Advanced**. The advanced configuration window displays. 
     <Lightbox src="scaleway_windows_ipfo2.webp" alt="" />
-    
 7. Click **Add**. Then enter the configuration details for the failover IP. Finally click **Add**.
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/configure-windows.mdx:31]:
```suggestion
7. Click **Add**. Then enter the configuration details for the failover IP. Finally, click **Add**.
```

File: dedibox-network/ip-failover/how-to/configure-windows.mdx [hunk 6]
```diff
@@ -21,18 +21,15 @@ This page shows you how to configure a [failover IP](/dedibox-network/ip-failove
 </Message>
 
 1. Log into your Dedibox using the **Remote Desktop** client.
-2. Open the start menu and type **Ethernet settings**. The Ethernet window displays. 
+2. Open the start menu and type **Ethernet settings**. The Ethernet window displays.
     <Lightbox src="scaleway_windows_ipfo1.webp" alt="" />
-3. Click **Change adatper options**. The network connections window displays. 
+3. Click **Change adatper options**. The network connections window displays.
 4. Right click on the Internet connection. Then click **Properties** in the pop-up menu.
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/configure-windows.mdx:27]:
```suggestion
4. Right-click on the Internet connection. Then click **Properties** in the pop-up menu.
```

File: dedibox-network/ip-failover/how-to/configure-windows.mdx [hunk 7]
```diff
@@ -21,18 +21,15 @@ This page shows you how to configure a [failover IP](/dedibox-network/ip-failove
 </Message>
 
 1. Log into your Dedibox using the **Remote Desktop** client.
-2. Open the start menu and type **Ethernet settings**. The Ethernet window displays. 
+2. Open the start menu and type **Ethernet settings**. The Ethernet window displays.
     <Lightbox src="scaleway_windows_ipfo1.webp" alt="" />
-3. Click **Change adatper options**. The network connections window displays. 
+3. Click **Change adatper options**. The network connections window displays.
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/configure-windows.mdx:26]:
```suggestion
3. Click **Change adapter options**. The network connections window displays.
```

File: dedibox-network/ip-failover/how-to/configure-reverse-dns.mdx [hunk 8]
```diff
@@ -5,9 +5,9 @@ meta:
 content:
   h1: How to the reverse DNS of a failover IP
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/configure-reverse-dns.mdx:6]:
```suggestion
  h1: How to configure the reverse DNS of a failover IP
```

Comment (jcirinosclwy) [reply dedibox-network/ip-failover/how-to/configure-reverse-dns.mdx:6]: I'm not sure whether 'configure' was missing, or it was something else.

File: dedibox-network/ip-failover/how-to/create-virtual-mac.mdx [hunk 9]
```diff
@@ -35,42 +35,43 @@ If you start a virtual machine in bridge mode, protection will be activated on t
...
 
 Once the virtual MAC is activated, use it to configure the network of your VM.
 
 ## How to delete a virtual MAC address
 
-You can remove the virtual MAC address associated to a failover IP if you do not require it any longer. 
+You can remove the virtual MAC address associated to a failover IP if you do not require it any longer.
 
 <Message type="tip">
   When you choose to delete a virtual MAC address on an IP address that shares its virtual MAC with other IPs, only the link between this specific IP address and the MAC is deleted, the other IP addresses are not updated.
```
Comment (jcirinosclwy) [inline dedibox-network/ip-failover/how-to/create-virtual-mac.mdx:65]: This sentence sounds a bit long and confusing, to me. 

Would something like this make sense? 

`If you delete a virtual MAC address that is shared by different IPs, this action will only affect the selected MAC address and IP. Any other IPs will remain as they are.`

File: dedibox-network/ip-failover/how-to/assign-failover-ip.mdx [hunk 10]
```diff
@@ -16,19 +16,19 @@ categories:
 Failover IP addresses allow you to have several IPs on the server and thus, for example, to have one IP for each website or for each virtual machine.
 Your IP failover can be routed at any time to your servers in just a few minutes by using the [Dedibox Console](https://console.online.net) or through the [Dedibox API](https://console.online.net/en/api/). By switching your failover IP addresses from one server to another, the hosted services remain available without reconfiguring the IP address.
```
Comment (nerda-codes) [inline dedibox-network/ip-failover/how-to/assign-failover-ip.mdx:17]:
```suggestion
Your IP failover can be routed at any time to your servers in just a few minutes by using the [Dedibox console](https://console.online.net) or through the [Dedibox API](https://console.online.net/en/api/). By switching your failover IP addresses from one server to another, the hosted services remain available without reconfiguring the IP address.
```

File: dedibox-network/ip-failover/how-to/assign-failover-ip.mdx [hunk 11]
```diff
@@ -16,19 +16,19 @@ categories:
...
 
-For each server, you can add failover IP address(es) according to the offer and your service level. 
+For each server, you can add failover IP address(es) according to the offer and your service level.
```
Comment (nerda-codes) [inline dedibox-network/ip-failover/how-to/assign-failover-ip.mdx:19]:
```suggestion
For each server, you can add failover IP addresses according to the offer and your service level.
```

File: dedibox-network/ip-failover/how-to/configure-reverse-dns.mdx [hunk 12]
```diff
@@ -16,22 +16,22 @@ categories:
 Failover IP addresses allow you to have several IP addresses with your server. This allows you to assign IP addresses to different websites or to virtual machines.
 Your IP failover can be routed at any time to your servers in just a few minutes by using our Account Management Console or through our API. By switching your failover IP addresses from one server to another, the hosted services remain available without reconfiguring the IP address.
```
Comment (nerda-codes) [inline dedibox-network/ip-failover/how-to/configure-reverse-dns.mdx:17]:
```suggestion
Your IP failover can be routed at any time to your servers in just a few minutes by using our Account Management console or through our API. By switching your failover IP addresses from one server to another, the hosted services remain available without reconfiguring the IP address.
```

File: dedibox-network/ip-failover/how-to/create-virtual-mac.mdx (hunk 9 above)
Comment (bene2k1) [inline dedibox-network/ip-failover/how-to/create-virtual-mac.mdx:65]:
```suggestion
  If you delete a virtual MAC address that is shared by different IPs, this action will only affect the selected MAC address and IP. Any other IPs will remain as they are.
```

**Pull request merged**